.7_create_iptables_entries/CUSTOM-ACCEPT.config

```diff
@@ -23,6 +23,8 @@
 -A CUSTOM-ACCEPT -j ACCEPT -s dev-master1.vocon-it.com -m comment --comment "dev-master1.vocon-it.com"
 -A CUSTOM-ACCEPT -j ACCEPT -s dev-node1.vocon-it.com -m comment --comment "dev-node1.vocon-it.com"
 -A CUSTOM-ACCEPT -j ACCEPT -s dev-node2.vocon-it.com -m comment --comment "dev-node2.vocon-it.com"
+-A CUSTOM-ACCEPT -j ACCEPT -s master1.prod.vocon-it.com -m comment --comment "master1.prod.vocon-it.com"
```

Comment:
PROD firewall entries need to be removed for dev, since we do not want to open the firewalls from prod to dev.